library/src/scala/quoted/Toolbox.scala

```diff
@@ -32,7 +33,7 @@ object Toolbox {
   }
 
   /** Setting of the Toolbox instance. */
-  class Settings private (val outDir: Option[String], val showRawTree: Boolean, val compilerArgs: List[String], val color: Boolean)
+  case class Settings private (val outDir: Option[String], val showRawTree: Boolean, val compilerArgs: List[String], val color: Boolean)
```

Comment:
This change makes it easy to debug what is wrong when printing -- it took me a while to figure out a silly problem related to color printing.

Reply:
You could remove the `val`s of the parameters